docs/source/hardening_macros.md

```diff
@@ -219,7 +206,7 @@ A final summary report is produced by default as `<run-path>/reports/metrics.csv
 
 A final manufacturability report is produced by default as `<run-path>/reports/manufacturability_report.csv`, this report contains the magic DRC, the LVS, and the antenna violations summaries.
 
-The final GDS-II should be found under `<run-path>/results/magic/`.
+The final GDS-II should be found under `<run-path>/results/signoff/`.
```

Comment:
or is it `results/final`?

Reply:
Either is fine but results/final/gds is more accurate.